test/verify/check-terminal

```diff
@@ -161,6 +161,25 @@ PROMPT_COMMAND='printf "\\033]0;%s@%s:%s\\007" "${USER}" "${HOSTNAME%%.*}" "${PW
         # Wait for text to show up
         wait_line(n + 2, "foo")
 
+        # Check that color theme changes
+        dark_style = "background-color: rgb(0, 43, 54);"
+        light_style = "background-color: rgb(253, 246, 227);"
+
+        b.select_from_dropdown("#theme-select", "Dark")
+        b.wait_attr(".xterm-viewport", "style", dark_style)
+
+        b.select_from_dropdown("#theme-select", "Light")
+        b.wait_attr(".xterm-viewport", "style", light_style)
+
+        # Reload and white-style should be remembered
+        # Reload on different page, as reloading on terminal prompts asking if you want to leave the site
```

Comment:
You can also just log out and back in, which is a more realistic and demanding test.

Reply:
Ok, I changed it to relogin, but still needs to do it on different page, as on terminal page I am asked if I want to leave the page. (interesting fact - This happens only in test, normally on logout I am not asked is I want to leave the page). It is most likely possible to do even on the same page, but I think it is good enough to switch page.